pwiz_tools/Shared/ProteowizardWrapper/MsDataFileImpl.cs

```diff
@@ -594,7 +594,7 @@ protected virtual SpectrumList SpectrumList
                             {
                                 if (GetMsLevel(spectrum) == 1)
                                 {
-                                    var function = MsDataSpectrum.WatersFunctionNumberFromId(id.abbreviate(spectrum.id), HasCombinedIonMobilitySpectra);
+                                    var function = MsDataSpectrum.WatersFunctionNumberFromId(id.abbreviate(spectrum.id), spectrum.id.Contains(MERGED_TAG));

```

Comment:
Hmm, maybe `HasCombinedIonMobilitySpectra && spectrum.id.Contains(MERGED_TAG)`. Because a lot of merged spectra won't be 3-array IMS spectra.

Reply:
Makes sense, done